stockflux-components/src/components/ScrollWrapperY/ScrollWrapperY.css

```diff
@@ -0,0 +1,14 @@
+.scroll-wrapper-y {
+  flex: 1 1 auto;
+  position: relative;
+  overflow-y: hidden;
+  overflow-x: hidden;
+  -webkit-transition: 0.6s all;
+  transition: 0.6s all;
```

Comment:
Can this be more restrictive than `all`?

Reply:
Yes indeed, here's the fix -> 08cff915e0c7c162c3c71c72e9adb0e02ba1de6c